master/reference/node/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment | Description                              | Schema | Default |
+| ------------- | -------- | ------ | ----- |
+| NODENAME | Node ID. | string | |
+| NO_DEFAULT_POOLS | Prevents calico from creating a default pool if one does not exist. | boolean | false |
+| HOSTNAME | Hostname. | string | |
+| IP | IP address which calico will bind to. | ip address | 10.0.0.1 |
```

Comment:
If an IP address is specified here, the address is written in to the node resource configuration, and will override any value previously configured.

If no IP address is specified, and an address has not yet been configured in the node resource, we will auto-detect an IPv4 address and configure the node resource with that address.  This means, if the node restarts and no --IP option is specified then the previously auto detected value will be used.

If a value of "autodetect" is specified, the IP address will be auto detected every time the node starts.  This will overwrite any value configured in the node resource.